.travis.yml

```diff
@@ -18,10 +18,3 @@ script:
 
 after_success:
 - codecov
```

Comment:
Something important seems to be missing here. Please double check!

Reply:
It's ok like this.